charts/spv-wallet-stack/charts/spv-wallet/values.yaml

```diff
@@ -123,22 +124,29 @@ config:
   debug_profiling: true
   # enable (ITC) incoming transaction checking
   disable_itc: true
-  nodes:
-    # deployment id used annotating api calls in XDeployment-ID header - this value will be randomly generated if not set
-    deployment_id: ''
+  
+  arc:
+    url: https://arc.taal.com
+    token: mainnet_06770f425eb00298839a24a49cbdc02c
+    _deployment_id: spv-wallet-deployment-id 
```

Comment:
I'm just wondering if we need to hold those "commented out" (underscored) params.
I'm not very familiar with this repository - so, just a thought.

Reply:
no we don't need it especially as previously there was such an entry and we should keep this solution:
```yaml
   # deployment id used annotating api calls in XDeployment-ID header - this value will be randomly generated if not set
    deployment_id: ''
```